client/data/hosting/use-site-logs-query.ts

```diff
@@ -127,6 +134,13 @@ function areRequestParamsEqual( a: SiteLogsParams, b: SiteLogsParams ) {
 		a.start === b.start &&
 		a.end === b.end &&
 		a.sortOrder === b.sortOrder &&
-		a.pageSize === b.pageSize
+		a.pageSize === b.pageSize &&
+		areFilterParamsEqual( a.filter, b.filter )
 	);
 }
+
```

Comment:
I am trying to understand what this function is doing exactly. I _think_ we are checking if the current filter parameters are the same as the previous filter parameters and if they are equal, we don't fetch the data again. Is that right?

Reply:
Yes, it seems right.